src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -230,6 +231,16 @@ private void applyFilterExpression(FilterExpression filter) {
 
     protected abstract void stopLoadingAnimationIfApplicable(AppliedFilterEvent e);
 
+    private void showErrorStatus(FilterExceptionEvent e) {
+        if (e.panel != this) return;
```

Comment:
Never mind, it is necessary as Parser.parse might throw a ParseException which is a subclass of FilterException.